support/DevEnvironments/.devcontainer_arm64/cmake-kits.json

```diff
@@ -0,0 +1,20 @@
+[
+    {
+        "name": "Default",
+        "compilers": {
+            "C": "/usr/bin/clang-10",
+            "CXX": "/usr/bin/clang++-10",
+            "Fortran": "/usr/bin/gfortran-9"
+        },
+        "cmakeSettings": {
+            "CHARM_ROOT": "/work/charm_7_0_0/multicore-linux-arm8-gcc",
```

Comment:
Is this line the only reason why the `.devcontainer` has to be duplicated? You can just delete this line because `CHARM_ROOT` is set as an environment variable. Then delete the separate `.devcontainer`.

Reply:
Yes, this is the only reason, now that I know that the docker file itself will have the same name on both platforms. I'll delete the other container and delete this line